config/system/alt.yaml

```diff
@@ -30,3 +30,12 @@ run_trigger:
   vlan_start: 1001
   vlan_end: 1009
   egress_vlan: 121
+
+mininet_subnet:
```

Comment:
What's this "for"?  "mininet" isn't meaningful unless you know the internals (breaks abstraction).  I'm honestly trying to remember myself what this is used for... Maybe `internal_subnet` -- just to differentiate from `external_subnets`?